testenv/testenv_arm32.properties

```diff
@@ -1,13 +1,14 @@
-TKG_REPO=git@github.com:adoptium/TKG.git
+TKG_REPO=https://github.com/adoptium/TKG.git
 TKG_BRANCH=master
-OPENJ9_REPO=git@github.com:eclipse-openj9/openj9.git
+OPENJ9_REPO=https://github.com/eclipse-openj9/openj9.git
 OPENJ9_BRANCH=master
-STF_REPO=git@github.com:adoptium/STF.git
+STF_REPO=https://github.com/adoptium/STF.git
 STF_BRANCH=master
-OPENJ9_SYSTEMTEST_REPO=git@github.com:eclipse-openj9/openj9-systemtest.git
+OPENJ9_SYSTEMTEST_REPO=https://github.com/eclipse-openj9/openj9-systemtest.git
 OPENJ9_SYSTEMTEST_BRANCH=master
-ADOPTOPENJDK_SYSTEMTEST_REPO=git@github.com:adoptium/aqa-systemtest.git
+ADOPTOPENJDK_SYSTEMTEST_REPO=https://github.com/adoptium/aqa-systemtest.git
 ADOPTOPENJDK_SYSTEMTEST_BRANCH=master
 JDK8_REPO=https://github.com/adoptium/aarch32-jdk8u
```

Comment:
Could we use the standard https format here? https://github.com/adoptium/aarch32-jdk8u.git

Reply:
updated